core/templates/pages/donate-page/donate-page.component.html

```diff
@@ -83,10 +83,28 @@
                 </div>
               </div>
             </div>
-            <div class="donate-our-learners">
-              <!--Add the learners heading and carousel here-->
-              <p></p>
-              <div class="learner-carousel"></div>
+            <div class="donate-learners-container">
+              <p> {{ 'I18N_DONATE_PAGE_CONTENT_HEADING_4' | translate }} </p>
+              <div class="donate-learners-bg">
+                <div class="donate-learners-carousel">
+                  <div class="donate-learners-tile" *ngFor="let learner of learners;index as id" id="tile-{{id}}">
+                    <picture>
+                      <source type="image/png" [srcset]="getStaticImageUrl(learner.imageUrl + '.png')">
```

Comment:
Could we just include the .png in learner.imageUrl so that we don't need to do logic/manipulation within the DOM template?